koku/cost_models/cost_model_manager.py

```diff
@@ -98,14 +97,14 @@ def update_provider_uuids(self, provider_uuids):
             except Provider.DoesNotExist:
                 LOG.info(f"Provider {provider_uuid} does not exist. Skipping cost-model update.")
             else:
-                queue = OCP_QUEUE if provider.type == Provider.PROVIDER_OCP else None
+                queue_choice = OCP_QUEUE if provider.type == Provider.PROVIDER_OCP else UPDATE_COST_MODEL_COSTS_QUEUE
```

Comment:
This can be where the `PRIORITY_QUEUE` is used, instead of `UPDATE_COST_MODEL_COSTS_QUEUE`